bittorrent/player.go

```diff
@@ -1217,7 +1217,12 @@ func (btp *Player) SetSubtitles() {
 
 		for _, f := range btp.t.files {
 			if strings.Contains(f.Path, currentPath) && util.HasSubtitlesExt(f.Path) {
-				collected = append(collected, util.GetHTTPHost()+"/files/"+f.Path)
+				escapedPath := f.Path
```

Comment:
There is a proper helper function for that:

`util.EncodeFileURL(btp.chosenFile.Path)`.

`f.Path` is a path, like it's written in torrent file, on Windows, it will be splitted by "\", on linux by "/".

Reply:
@elgatito 
thanks, fixed